src/schema/v2/partnerOffer.ts

```diff
@@ -25,6 +31,28 @@ export const PartnerOfferType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       resolve: ({ partner_id }) => partner_id,
     },
+    priceListedMessage: {
+      type: GraphQLString,
```

Comment:
I see you initially went with this type as a `Money` interface then changed your mind. Is there a specific reason for that?

Reply:
@MounirDhahri sure! I've just noticed that Artwork details component uses `saleMessage` field from Metaphysics, so I've decided to stick to the same approach. `saleMessage` is a formatted string 

- saleMessage field resolver in MP: https://github.com/artsy/metaphysics/blob/ec13f424f9547d2961f2d71a6ff1f920a14ebbb9/src/schema/v2/artwork/index.ts#L1501
- place where gravity queries it: https://github.com/artsy/force/blob/main/src/Components/Artwork/Details.tsx#L376

Reply:
You think it's worth re-doing it with Money?